src/com/tekion/cricket/Match.java

```diff
@@ -0,0 +1,267 @@
+package com.tekion.cricket;
+
+import java.util.Scanner;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Match {
+    private final Team team1;
+    private final Team team2;
+    enum Winner{
```

Comment:
pls separate out this enum into separate file

Reply:
done, I'll push the changes in the week-2 branch.